src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -431,7 +432,7 @@ protected function getDoctrine()
     /**
      * Get a user from the Security Token Storage.
      *
-     * @return object|null
+     * @return UserInterface|object|null
```

Comment:
We could do the same in `TokenInterface` and add the description from `TokenInterface` here too:

> Can be a UserInterface instance, an object implementing a __toString method, or the username as a regular string